charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```
@@ -135,6 +135,10 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+upgradeHooks:
```

These are not just upgrade hooks. They are a combination of pre-delete and post-install and post-upgrade hooks.

```suggestion
helmHooks:
```